src/poly/poly.jl

```diff
@@ -878,17 +878,51 @@ function derivative(x::spoly, v::spoly)
 end
 
 @doc Markdown.doc"""
-   jacobi(x::spoly)
-> Given a polynomial $x$ this function the Jacobian ideal of $x$.
+   jacobian_ideal(x::spoly)
+> Given a polynomial $x$ this function returns the Jacobian ideal of $x$.
 """
-function jacobi(p::spoly)
+function jacobian_ideal(p::spoly)
```

Comment:
Are you sure you really want this in poly.jl. Yes, it takes a poly as input, but if you only had everything up to polynomials, you couldn't have this function. So it seems like it belongs at the level of ideals.

It's entirely up to you, and depends on how much you see this as a property of polynomials, versus a constructor of a kind of ideal. But it's certainly best to stick it where you think people are most likely to look for it.

Reply:
I agree that it does not fit here, but it also does not fit into ideal.jl, since the general case of jacobian ideals (non principal ideals) needs minors of matrices. For the time being I would let it here. 
Once we have minors of matrices (probably my next pull request, if no one works one it), I would propose to create a "Singularity.jl" which features the singularity theory related functions, similar to "resolutions.jl".